### PR TITLE
Add armel-tizen

### PR DESCRIPTION
**Pull request by clamp03**
- Add armel-tizen to manifest.json
- Update Dockerfile of crossdeps to install libxml2-utils, cpio and rpm2cpio
- Add Dokerfile and hooks for armel-tizen
- armel-tizen doesn't make /bin directory
  Update directory existence check in build-rootfs.sh
  from /rootfs/$arch/bin to /rootfs/$arch/usr/bin
- Issue: https://github.com/dotnet/runtime/issues/2394

### CODE REVIEW

File: src/ubuntu/18.04/crossdeps/Dockerfile [hunk 1]
```diff
@@ -27,6 +27,9 @@ RUN apt-get update \
         libarchive-dev \
         libbsd-dev \
         libmpc-dev \
+        libxml2-utils \
+        cpio \
+        rpm2cpio \
```
Comment (mthalman) [inline src/ubuntu/18.04/crossdeps/Dockerfile:32]: Please list the new packages in alphabetical order with the existing package listing.

File: src/ubuntu/build-scripts/build-rootfs.sh [hunk 2]
```diff
@@ -49,12 +49,12 @@ if [ $? -ne 0 ]; then
...
 
-echo "Checking existence of /rootfs/$arch/bin"
+echo "Checking existence of /rootfs/$arch/usr/bin"
 docker exec $buildRootFSContainer \
-    [ -d /rootfs/$arch/bin ]
+    [ -d /rootfs/$arch/usr/bin ]
 
 if [ $? -ne 0 ]; then
-    echo "Rootfs build failed: rootfs/$arch/bin empty"
+    echo "Rootfs build failed: rootfs/$arch/usr/bin empty"
```
Comment (mthalman) [inline src/ubuntu/build-scripts/build-rootfs.sh:57]: It looks like these changes cause the build to fail for ubuntu/18.04/cross/freebsd/11.  I would recommend defining a variable for the bin path that defaults to `/rootfs/$arch/bin` but can be overridden by a parameter to this script. And then override it in the src/ubuntu/18.04/cross/armel-tizen/hooks/pre-build script by passing `rootfs/armel/usr/bin`.

File: src/ubuntu/18.04/crossdeps/Dockerfile (hunk 1 above)
Comment (clamp03) [reply src/ubuntu/18.04/crossdeps/Dockerfile:32]: Thank you for the review I will fix it.

File: src/ubuntu/build-scripts/build-rootfs.sh (hunk 2 above)
Comment (clamp03) [reply src/ubuntu/build-scripts/build-rootfs.sh:57]: Thank you for the review I will fix it.

**Review by mthalman — APPROVED** [review]

**Comment by clamp03** [comment]
@mthalman Thank you.

**Pull request merged**